modules/motionmatch/animation_motion_match_editor.cpp

```diff
@@ -0,0 +1,285 @@
+#include "animation_motion_match_editor.h"
+#include <iostream>
+
+#ifdef TOOLS_ENABLED
+#include "core/io/resource_loader.h"
+
+bool AnimationNodeMotionMatchEditor::can_edit(
+    const Ref<AnimationNode> &p_node) {
```

Comment:
Just a note, in the cpp files, we use \<tab\> instead of \<space\> for indentation.

Reply:
I actually set up the pre-commit-hook too. Will recheck it..

Reply:
Nice.